docs/contracts/visibility-and-getters.rst

```diff
@@ -6,15 +6,32 @@
 Visibility and Getters
 **********************
 
-Solidity knows two kinds of function calls: internal
```

Comment:
I think we should not delete this paragraph, it explains what internal and external calls are and that is important for what follows.

Reply:
The reason why I decided to remove this paragraph is (1) because external / internal function calls are briefly described in visibility descriptions (e.g. in `external` function description) and (2) because of `Because of that, there are four types of visibility for functions and state variables.` sentence - it is not obvious why two call types produce four visibility types.

But looking with a fresh eye I agree that we should provide some note on external / internal function calls. It'll be useful for newcomers. I see at least two options here: (1) we can leave only the first sentence or (2) we can provide more information to explain how two call types produce four visibility types.

Reply:
I've returned the paragraph and rephrased it a bit at 1fed766.